public/assets/sass/variables.scss

```diff
@@ -0,0 +1,11 @@
+$bg-color: #f2f2f2;
+$primary-text-color: #555555;
+$text-color: #ffffff;
+$titles-font-color: #333333;
+$primary-color-orange: #f85c37;
+$accent-color: #7b7b7b;
+$text-color-black: #2f2f2f;
```

Comment:
@wingchunkf Instead of giving a variable name a color like "-black" I would steer towards a more friendly name in the event we want to change that variable value in the future, ie: what if we change "text-color-black" to an orange or green color? I would lean more towards expressing a variables color with words like "primary", "secondary", "highlight" or "accent".

Reply:
Thank you for the information. I will follow those guidelines in the future.